spec/features/game_spec.rb

```diff
@@ -0,0 +1,39 @@
+require './lib/rps'
+
+feature '/game' do
+  scenario 'you see your name' do
+    visit '/'
+    fill_in 'player', with: 'Dom'
+    click_button 'go!'
```

Comment:
There's a small amount of repeated code here in game_spec.rb: 
    visit '/'
     fill_in 'player', with: 'Dom'
     click_button 'go!' (line 5 - 7)
     I would have made this into a separate function called 'visit_home_and_click_go' or something so that you don't have to repeat those lines in your other tests, but I don't know if this would break your test so maybe you tried it already? Otherwise, all of your code is really clean and easy to read - I like it 👍